io.github.Hexchat.Plugin.Perl.json

```diff
@@ -3,7 +3,7 @@
   "branch": "20.08",
```

Comment:
branch should be updated to `21.08`.

The main `io.github.Hexchat` package needs to have this set as the branch name for the extension.

Reply:
https://github.com/flathub/io.github.Hexchat/commit/736e3ce05f4d61046b2541b8e4c46fba25961180

Reply:
Thanks.